modules/apps/digital-signature/digital-signature-web/test/js/pages/DigitalSignature.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {act, render} from '@testing-library/react';
+import React from 'react';
+
+import DigitalSignature from '../../../src/main/resources/META-INF/resources/js/pages/DigitalSignature';
+
+const initialState = {
+	allowedFileExtensions: '',
+};
+
+describe('DigitalSignature', () => {
+	beforeEach(() => {
+		jest.useFakeTimers();
+	});
+
+	it('renders', async () => {
+		const {asFragment} = render(<DigitalSignature {...initialState} />);
```

Comment:
Since we have just one attribute in initialState, you could use allowedFileExtensions directly, another point: this same attribute is an array and not string

```<DigitalSignature allowedFileExtensions={["csv", "pdf"]}  />```